src/GafferImage/DisplayTransform.cpp

```diff
@@ -101,12 +101,6 @@ void DisplayTransform::hashTransform( const Gaffer::Context *context, IECore::Mu
 	std::string display = displayPlug()->getValue();
 	std::string view = viewPlug()->getValue();
 
-	if( colorSpace.empty() || display.empty() || view.empty() )
```

Comment:
I don't think we should be removing this. I'm pretty sure it's the reason for this failing test case :

```
======================================================================
FAIL: testImageHashPassThrough (GafferImageTest.DisplayTransformTest.DisplayTransformTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/GafferHQ/gaffer/install/gaffer-0.31.0.0-linux/python/GafferImageTest/DisplayTransformTest.py", line 121, in testImageHashPassThrough
    self.assertEqual( i["out"].imageHash(), o["out"].imageHash() )
AssertionError: IECore.MurmurHash( "2a28224927469f228823124cabdd0e8d" ) != IECore.MurmurHash( "d427f1dc196e776c84e9b8a642b31096" )
----------------------------------------------------------------------
```